website/docs/plugins/signing.html.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: "registry"
+page_title: "Plugin Signing"
+sidebar_current: "docs-plugins-signing"
+description: |-
+  Terraform plugin signing trust levels
+---
+
+# Plugin Signing
+
+There are four levels of signing and programattic authentication for plugins in Terraform:
+
+* **Signed by HashiCorp** - are built, signed, and supported by HashiCorp. Official provider plugins can 
+typically be used without specifying provider source information in your Terraform configuration.
```

Comment:
I'm not sure why we're conflating the optional use of provider source attributes with the meaning of the signature. It seems clearer to me to omit this altogether here (and below).

Reply:
this was just a copy and paste of copy i wrote for the registry, will amend